src/OnChain.ts

```diff
@@ -100,14 +100,15 @@ export const OnChainMixin = (superclass) =>
     }
 
     // amount in sats
-    async onChainPay({ address, amount, memo }: IOnChainPayment): Promise<ISuccess> {
+    async onChainPay({ address, amount, memo, sendAll = false }: IOnChainPayment): Promise<ISuccess> {
```

Comment:
Should a check comparing the balance.total_in_BTC vs amount be added when sendAll? Or amount stops being a relevant parameter in this case?

Reply:
Added comments where 'amount' is still used, which should be affected if is sendAll and amount is not specified or is not relevant.